pubspec.yaml

```diff
@@ -11,7 +11,8 @@ environment:
 
 dependencies:
   crypto: ^3.0.3
-  dio: ^5.2.0
+  intl: ^0.18.1
```

Comment:
We discussed this new dependency in a private channel and decided to leave it as a dev-only dependency for matrix tests. 

My concern was that a new package dependency was introduced only for the number formatting in the evaluation logs. We decided to keep Dart's basic number formatting here. (e.g.: whole decimal numbers will get `.0`)

Reply:
> We decided to keep Dart's basic number formatting here. (e.g.: whole decimal numbers will get .0)

I'd prefer to stick to the spec if possible. If it can't be done otherwise, we can still fix up the format using e.g. a simple regex like this:

```dart
s = s.replaceAllMapped(RegExp(r'\.0+(e|$)'), (m) => m[1]!);
```